content/docs/user-guide/packaging/project-export-cli.md

```diff
@@ -0,0 +1,271 @@
+---
+linkTitle: Project Export CLI
+title: Project Export CLI Tool
+description: Learn how to use the Project Export CLI to automate preparing your project for release.
+toc: true
+weight: 400
+---
+The export-project CLI command is used to automatically package game code and assets for a release.
+
+{{< note >}}
+If you wish to learn how to prepare projects manually for Windows, please consult the page [Creating a Project Game Release Layout for Windows.](/docs/user-guide/packaging/windows-release-builds)
+{{< /note >}}
+
+## Prerequisites
+To make best use of the `export-project` command, it is recommended to have a project with at least one starting scene, and all necessary seedlist files prepared. AssetProcessor Registry file settings may need to be tweaked. To learn how to set this up, please consult the following page: [Creating a Project Game Release Layout for Windows, section: Set the Starting Level](windows-release-builds/#set-the-starting-level).
+
+To learn more about the AssetBundler and Seed Files, please visit the [overview page on the AssetBundler tool](https://docs.o3de.org/docs/user-guide/packaging/asset-bundler/overview/).
+
+## Getting Started
+To use the `export-project` command correctly, it is important to first understand how it works at a high level.
+
+The command operates in 2 stages:
+1. Script Loading
+2. Script Running
+
+### Script Loading
+Before the export process can begin, the export-project command does the following pre-processing steps:
+* Parse and validate initial set of arguments, and filters arguments to be processed by desired export script
+* Load the export script as a python module
+* Validate the expected project path
+* Modify the system path with O3DE CLI Python API so that user scripts can easily import API code and modules
+* Prepares the O3DE script context, which contains useful values like project path, engine path, project name, CMake args, etc.
+* Runs the export script with the prepared context injected
+
+### Script Running
+The `export-project` command can run any python script that the user designates as the export script. As a result, the exact behavior entirely depends on the supplied script. O3DE includes a standard script, which is described [further below.](#standard-export-script)
+
+Any script run by the command has access to the running context of the exporter, along with common APIs.
+
+## Export CLI Command
+The `export-project` command is accessed via the O3DE CLI that is shipped with the engine. This is found in `<engine>/scripts/o3de.bat`, or on Unix systems as `<engine>/scripts/o3de.sh`. The command has the following arguments:
+
+| Argument Name | Description | Required? |
+| - | - | - |
+| `--export-script` | Choose an external python script to run. The path can be relative or absolute. | yes |
```

Comment:
Out of scope for this since this is documentation, but if the user didn't supply this could we just use the default script that comes with the engine? And from the project path might be able to tell if it is engine or project centric?

Reply:
Yes that would be possible, but it would require updating the export scripts themselves first. Once that is done I will update the documentation accordingly